src/utils/dbUtils.ts

```diff
@@ -106,6 +106,8 @@ export const saveToDb = (
     })
 }
 
+export const wrappedSaveToDb = (docs: DbDoc[]): void => saveToDb(dbRates, docs)
```

Comment:
This works for now but if you start wrapping multiple items, you might consider using a pattern similar to this: https://github.com/EdgeApp/edge-core-js/blob/master/src/util/file-helpers.js